prow/cmd/deck/static/pr-script.js

```diff
@@ -694,15 +694,33 @@ function createMergeLabelCell(labels, notMissingLabel = false) {
     return cell;
 }
 
-function processLabelName(label) {
-    label = label.split(" ").join("");
-    if (label.startsWith("area/")) {
-        return "area";
-    } else if (label.startsWith("size/")) {
-        return label.slice(5);
-    } else {
-        return label;
+/**
+ * escapeLabel escaped label name that returns a valid name used for css
+ * selector.
+ * @param {string} label
+ * @returns {string}
+ */
+function escapeLabel(label) {
```

Comment:
I'll trust that you tested it well, it looks more or less like what I'd expect but I have not been writing all that much javascript lately :-)